mandatory/3-playing-computer.js

```diff
@@ -6,13 +6,13 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). b is undefined for console.log(b)
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  4. How many times is "f1" called? 2 times
+  5. How many times is "f2" called? 3 times
+  6. What value does the "a" parameter take in the first "f1" call? 8
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? 6
```

Comment:
I think here is 8 as well

Reply:
honestly I'm not sure about this one yet